### PR TITLE
update ci to 0.7.25

### PR DESCRIPTION
**Pull request by pankore**
- CI use docker image 0.7.25

### CODE REVIEW

**Review by andy31415 — APPROVED** [review]

File: .github/workflows/examples-efr32.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
     if: github.actor != 'restyled-io[bot]'
 
     container:
-      image: connectedhomeip/chip-build-efr32:0.7.3
+      image: connectedhomeip/chip-build-efr32:0.7.23
```
Comment (jmartinez-silabs) [inline .github/workflows/examples-efr32.yaml:41]: EFR32 now requires 0.7.25 after #27618

**Review by jmartinez-silabs — CHANGES_REQUESTED** [review]

**Review by woody-apple — APPROVED** [review]

File: .github/workflows/examples-efr32.yaml (hunk 1 above)
Comment (pankore) [reply .github/workflows/examples-efr32.yaml]: updated, thanks

**Review by jmartinez-silabs — APPROVED** [review]

**Comment by woody-apple** [comment]
I think what you're looking for will be here?

https://github.com/project-chip/connectedhomeip/pull/27879

**Comment by andy31415** [comment]
This broke cloudbuild:  vscode 0.7.25 image does not seem to be in dockerhub (and we recently removed it towards gh actions)

**Pull request merged**